backend/src/test/java/com/htecgroup/skynest/service/impl/FileServiceImplTest.java

```diff
@@ -112,6 +116,37 @@ void getAllFilesWithParent() {
         .findAllByParentFolderId(fileMetadataEntity.getParentFolder().getId());
   }
 
+  @Test
+  void moveFileToFolderTest() {
+    FileMetadataEntity expectedFileMetadataEntity =
+        FileMetadataEntityUtil.getNotRootFileMetadataEntity();
+    when(fileMetadataRepository.findById(any()))
+        .thenReturn(Optional.of(expectedFileMetadataEntity));
+    FolderEntity folderEntity = FolderEntityUtil.getFolderWithoutParent();
+    when(folderRepository.findById(any())).thenReturn(Optional.of(folderEntity));
+    when(fileMetadataRepository.save(any())).thenReturn(expectedFileMetadataEntity);
+    fileService.moveFileToFolder(UUID.randomUUID(), UUID.randomUUID());
+
+    Assertions.assertNotNull(expectedFileMetadataEntity.getParentFolder());
+    verify(fileMetadataRepository, times(1)).findById(any());
```

Comment:
can you change instead of any() to check for the actual value? Same applies to every any() in verify methods